1.pre_process_data/scripts/0.fix_pathing.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook fixes issues with data paths by replacing spaces with underscores
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+
+# In[2]:
+
+
+# set data path
+data_path = pathlib.Path("../../data").resolve(strict=True)
```

Comment:
This looks like a line which might appear in many notebooks in this project. Consider creating a Python module which could provide this object to any notebook via an import (and consistent behavior when it comes to directory paths). Doing this might save the trouble of both adding the line in to every notebook and adjusting it when it comes to any nested file structures that may emerge.